### PR TITLE
Automation for firewall allocation

### PR DESCRIPTION
**Pull request by TravisFallsMSFT**
# Type of PR

- [x] Documentation changes
- [ ] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## PR Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My code follows the code style of this project.
- [ ] I ran lint checks locally prior to submission.
- [ ] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Does this introduce a breaking change

- [ ] YES
- [ ] NO

## Validation

- [ ] Unit tests updated and ran successfully
- [ ] Update documentation or issue referenced above

## Issues Closed or Referenced
Closes #1031

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,15 +7,17 @@
...
-        - [Azure ClI](#azure-cli)
+        - [Azure ClI Login](#azure-cli-login)
   - [Setup Infrastructure](#setup-infrastructure)
   - [Infrastructure Setup During Each Script](#infrastructure-setup-during-each-script)
     - [2-CreateHub.sh](#2-createhubsh)
     - [3-AttachSpokeAndClusterToHub.sh](#3-attachspokeandclustertohubsh)
   - [Deploying NGSA Applications](#deploying-ngsa-applications)
     - [🛑 Prerequisite - Setup Cosmos DB in secure baseline](#-prerequisite---setup-cosmos-db-in-secure-baseline)
     - [Create managed identity for NGSA app](#create-managed-identity-for-ngsa-app)
+    - [Create managed identity for ngsa-app](#create-managed-identity-for-ngsa-app-1)
```
Comment (AAkindele) [inline README.md:18]: This is a dup of the line above. Was it supposed to link somewhere else in the documentation?

File: README.md [hunk 2]
```diff
@@ -170,6 +172,58 @@ export ASB_NGSA_MI_RESOURCE_ID=$(az identity create -g $ASB_RG_CORE -n $ASB_NGSA
 
 ```
 
+### Create managed identity for ngsa-app
```
Comment (AAkindele) [inline README.md:175]: This section is also a dup of the section above.

File: README.md [hunk 3]
```diff
@@ -170,6 +172,58 @@ export ASB_NGSA_MI_RESOURCE_ID=$(az identity create -g $ASB_RG_CORE -n $ASB_NGSA
...
+### Assign User to Cosmos DB
+
+```bash
+  
+  az login --tenant 'XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX'
+  az account set --subscription 'XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX'
+
+  # Variables
+  # built in Cosmos DB roles
+  local roles=("00000000-0000-0000-0000-000000000002") #Array of built in Cosmos roles
```
Comment (AAkindele) [inline README.md:203]: Cosmos roles is new to me. Is there recommended documentation on how to find these role ids? Or is the one listed here the actual role id?

**Comment by TravisFallsMSFT** [comment]
Closing as duplicate.

**Pull request closed without merging**